lib/explorer/series.ex

```diff
@@ -2732,6 +2732,72 @@ defmodule Explorer.Series do
   @spec is_not_nil(Series.t()) :: Series.t()
   def is_not_nil(series), do: Shared.apply_impl(series, :is_not_nil)
 
+  # Float predicates
+
+  @doc """
+  Returns a mask of finite values.
+
+  ## Examples
+
+      iex> s1 = Explorer.Series.from_list([1, 2, 0, nil])
+      iex> s2 = Explorer.Series.from_list([0, 2, 0, nil])
+      iex> s3 = Explorer.Series.divide(s1, s2)
+      iex> Explorer.Series.is_finite(s3)
+      #Explorer.Series<
+        Polars[4]
+        boolean [false, true, false, nil]
+      >
+  """
+  @doc type: :element_wise
+  @spec is_finite(Series.t()) :: Series.t()
+  def is_finite(%Series{dtype: dtype} = series) when numeric_dtype?(dtype),
```

Comment:
Do they actually work with integers? Or floats only? If they work with integers, can you add a test to `series_test.exs` for at least one of them? Thank you! :heart:

Reply:
ah, I overlooked. It works only with float. Let me update.